script.js

```diff
@@ -39,13 +39,26 @@ class App extends React.Component {
 
   }
   signIn(email, password) {
-    // sign in APIs here
-    this.setState({
-      user: {
-        email,
-        password } });
-
-
+		axios.post('http://localhost:3000',{
+			username: email,
+			password: password
+		})
+		.then((response) => {
+			console.log(response);
+			if(response.data === 'Login succeeded'){this.setState({
```

Comment:
We should probably base our logic on HTTP response status codes rather than text messages which might change.

Reply:
Logic now based on status response codes